tests/parsely/test-mu-parsely-integration.php

```diff
@@ -63,13 +63,7 @@ public function test_bootstrap_modes() {
 	}
 
 	public function test_parsely_ui_hooks() {
-		global $parsely;
-
-		$expected = in_array( self::$test_mode, [ 'filter_enabled', 'filter_and_option_enabled' ] ) ? 10 : false;
-		$this->assertSame( $expected, has_action( 'admin_menu', array( $parsely, 'add_settings_sub_menu' ) ) );
-		$this->assertSame( $expected, has_action( 'admin_footer', array( $parsely, 'display_admin_warning' ) ) );
-		$this->assertSame( $expected, has_action( 'widgets_init', 'parsely_recommended_widget_register' ) );
```

Comment:
With the random test order, we cannot use these assertions anymore: WordPress will clean up the hooks set up by Parsely, and `has_action()` will return `false`.

Since wp-parsely does not offer a function to invoke to set up all necessary hooks (it runs `add_action()` [in the main file when it is included](https://github.com/Automattic/vip-go-mu-plugins/blob/364d486c99125bafbf94f82409aaeb0ad5ca573a/wp-parsely-2.6/wp-parsely.php#L38)), we cannot reliably check these actions.

Reply:
Ref: https://app.circleci.com/pipelines/github/Automattic/vip-go-mu-plugins/8681/workflows/71b71672-1369-405c-901a-0c704fd5d75b/jobs/77534?invite=true#step-105-2437